### DISCUSSION TITLE
Winlog fields missing

### DISCUSSION THREAD

**Discussion by nilssachs** [Security-Onion-Solutions/securityonion#11538] [2.4]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

upgrading

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

64

### Storage for /

500GB

### Storage for /nsm

3TB

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hey guys,

I am currently struggling with the integration of ElasticAgent. 
Logs are transferred, but all the relevant fields from the Windows-Integration (windows-endpoints) are not included in them (e.g. winlog and powershell fields).
<img width="663" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/12412595/f1db170b-be8b-4767-93ed-b8a437e07448">

Is there anything that needs to be activated to get this information?

Thanks a lot, Nils

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by defensivedepth** [comment]
Hello there!

Did you use the Elastic Agent installer from the `Downloads` page in the Security Onion Console?  Are you using the `endpoints-initial` Integration Policy? Did you modify anything on that policy?

Finally, what version of Windows did you install it on?

**Reply by nilssachs** [reply]
Hi Josh, thanks for your reply.
Installer is from Downloads, Policy is endpoints-initial. I removed the Elastic Defend integration and added it again, since that lead to an unhealthy status after the last upgrade. But Winlog was missing all the time.
Agent is running on Win Server 2019

**Reply by defensivedepth** [reply]
Double-check that you are looking at the correct logs shipped by Elastic Agent. Here is what you should be seeing for Windows Eventlogs (that SOC Source field in your screenshot is showing a generic index - it should be something like `$HOSTNAME:.ds-logs-system.application-default-2023.09.21-000002`):

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/954732/288b2315-1686-48c7-84f3-48b3b2f3ae73)
